src/modules/keyboardmanager/common/Helpers.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include <windows.h>
+#include <stdlib.h>

```

Comment:
Do you really need all these includes in the header file?  If possible your better off moving the includes to the .cpp file and `forward declaring` the types referenced.

Reply:
I've also noticed that having all these WinUI header files adds a lot to compilation time. Does that mean it would be better to move them to a pre-compiled header and include that directly on all the files?

Reply:
Yep, these would probably be good candidates for a pch.  I wouldn't expect them to change very often.